flocker/control/httpapi.py

```diff
@@ -362,13 +365,14 @@ def saved(ignored):
         return saving
 
     @app.route("/state/datasets", methods=['GET'])
-    @user_documentation("""
-        Get current cluster datasets.
```

Comment:
One though I had, is, that we could just turn this into a header here, i.e.
```python
@user_documentation("""
    Get current cluster datasets
    ============================

    ....
```
But given that I think we want to enforce headers being there, then it is easier to do what you did.